tests/main.nf.test

```diff
@@ -55,7 +51,7 @@ nextflow_pipeline {
             assert json.CSE.meta.single_end.equals(false)
             assert json.CSE.meta.merge.equals(false)
             assert json.CSE.meta.downsampled.equals(false)
-            
+
```

Comment:
In addition to the test fixes you've made, since this PR adds the ability to save a gzipped JSON report for each sample, I'm wondering if you could add an assert statement which verifies that this file exists in the IRIDA Next JSON output.

I cannot comment on the exact line in the PR (since you haven't made changes around it), but I'm wondering if you could make the listed changes near the following line in this file:

# Change 1

**below this line**
```groovy
assert iridanext_samples.CSE.findAll { it.path.contains("assembly/length_filtered_contigs") }.size() == 0
```
**add this line**
```groovy
assert iridanext_samples.CSE.findAll { it.path.contains("SummaryReport/CSE_flat_sample.json.gz") }.size() == 0
```

# Change 2

**below this line**
```groovy
assert iridanext_samples.short.findAll { it.path == "assembly/length_filtered_contigs/short_filtered.fasta.gz" }.size() == 1
```
**add this line**
```groovy
assert iridanext_samples.short.findAll { it.path == "SummaryReport/short_flat_sample.json.gz" }.size() == 1
```

Reply:
fixed in: 10dc13b8f2c4dea5879e38ffde983753ea047967